programs/whirlpool/src/util/token_2022.rs

```diff
@@ -149,7 +149,8 @@ pub fn initialize_token_metadata_extension<'info>(
     let token_mint_unpacked =
         StateWithExtensions::<spl_token_2022::state::Mint>::unpack(&token_mint_data)?;
     let new_account_len = token_mint_unpacked
-        .try_get_new_account_len::<spl_token_metadata_interface::state::TokenMetadata>(&metadata)?;
+        .try_get_new_account_len_for_variable_len_extension::<spl_token_metadata_interface::state::TokenMetadata>(&metadata)?;
```

Comment:
memo: breaking change on spl-token-2022-1.0.0 (0.9.0 -> 1.0.0)

Reply:
ah sorry, this is not change request. I believe your update is correct of course.
Just a memo for reviewers. 😅